Block/Form/Cc.php

```diff
@@ -124,15 +124,18 @@ public function getCcAvailableTypesByAlt()
 		$types = [];
 		$ccTypes = $this->adyenHelper->getAdyenCcTypes();
 
-		$availableTypes = $this->adyenHelper->getAdyenCcConfigData('cctypes');
-		if ($availableTypes) {
-			$availableTypes = explode(',', $availableTypes);
-			foreach (array_keys($ccTypes) as $code) {
-				if (in_array($code, $availableTypes)) {
-					$types[$ccTypes[$code]['code_alt']] = $code;
-				}
-			}
-		}
+        $enableAvailableTypes = $this->adyenHelper->getAdyenCcConfigData('enablecctypes');
+        if ($enableAvailableTypes) {
+            $availableTypes = $this->adyenHelper->getAdyenCcConfigData('cctypes');
+            if ($availableTypes) {
+                $availableTypes = explode(',', $availableTypes);
+                foreach (array_keys($ccTypes) as $code) {
+                    if (in_array($code, $availableTypes)) {
+                        $types[$ccTypes[$code]['code_alt']] = $code;
+                    }
+                }
+            }
+        }
```

Comment:
Add some early returns I think. It's easier to read.

```suggestion
        if (!$enableAvailableTypes) {
            return $types;
        }
     
        $availableTypes = $this->adyenHelper->getAdyenCcConfigData('cctypes');
        if (!$availableTypes) {
            return $types;
        }
                
        $availableTypes = explode(',', $availableTypes);
        foreach (array_keys($ccTypes) as $code) {
            if (in_array($code, $availableTypes)) {
                $types[$ccTypes[$code]['code_alt']] = $code;
            }
        }    
```